charts/incubator/passky-server/values.yaml

```diff
@@ -0,0 +1,103 @@
+image:
+  repository: rabbitcompany/passky-server
+  pullPolicy: IfNotPresent
+  tag: 8.1.6@sha256:ef803c3ffbabde6b79b996f5b33fcab0515f6439ccc961129d5da044c5518979
+securityContext:
+  container:
+    readOnlyRootFilesystem: false
+    runAsGroup: 0
+    runAsUser: 0
+service:
+  main:
+    ports:
+      main:
+        port: 18080
+        targetPort: 80
+workload:
+  main:
+    podSpec:
+      containers:
+        main:
+          env:
+            SERVER_LOCATION: "US"
+            SERVER_CORES: 1
+            ADMIN_USERNAME: "admin"
+            ADMIN_PASSWORD: ""
+            CF_TURNSTILE_SITE_KEY: "1x00000000000000000000AA"
+            CF_TURNSTILE_SECRET_KEY: "1x0000000000000000000000000000000AA"
+            DATABASE_ENGINE: mysql
+            MYSQL_HOST:
+              secretKeyRef:
+                expandObjectName: false
+                name: '{{ printf "%s-%s" .Release.Name "mariadbcreds" }}'
+                key: plainhost
+            MYSQL_PORT: 3306
+            MYSQL_DATABASE: "{{ .Values.mariadb.mariadbDatabase }}"
+            MYSQL_USER: "{{ .Values.mariadb.mariadbUsername }}"
+            MYSQL_PASSWORD:
+              secretKeyRef:
+                expandObjectName: false
+                name: '{{ printf "%s-%s" .Release.Name "mariadbcreds" }}'
+                key: mariadb-password
+            MYSQL_SSL: "false"
+            MYSQL_SSL_CA: "/etc/ssl/certs/ca-certificates.crt"
```

Comment:
Do we have to manually disable SSL on db and why we specify CA if its disabled?

Reply:
Removed both